app/containers/OldEvents/index.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { bindActionCreators } from 'redux'
+import { connect } from 'react-redux'
+import { scrollSpy, Events, animateScroll } from 'react-scroll'
+import * as actions from 'containers/Events/actions'
+
+import styles from './styles.css'
+import PublicEvent from 'components/PublicEvent'
+import PublicEventMenu from 'components/PublicEventMenu'
+
+export class OldEvents extends React.Component {
```

Comment:
Does this class need to be exported? Since the connected version is the one you'd wanna use.

Reply:
Very true 👍 Fixed.